src/streamlink_cli/argparser.py

```diff
@@ -307,10 +307,19 @@ def boolean(value):
     default=find_default_player(),
     help="""
     Player to feed stream data to. This is a shell-like syntax to
-    support passing options to the player. For example:
+    support using a specific player, for example:
+
+      "streamlink <url> --player=mpc-hc"
+    
+    Options may also be passed to the player. For example:
 
       "vlc --file-caching=5000"
 
+    Paths can also be passed via this option in the event the
+    player's path has not been set in the configuration file:
+
+      "streamlink <link> --player=path\to\mpc-hc64.exe"
```

Comment:
Good notes, I'll update it when I get home. @bastimeyer The quote requirement for spaces is already notated further down the page, it could be included in both though, thoughts?